postal-address.js

```diff
@@ -0,0 +1,70 @@
+const formatters = require('./formatters.js');
+
+/**
+ * This file is a translation of the `postal_address.py` python file from
+ * the [GOV.UK notifications-utils repo](https://github.com/alphagov/notifications-utils)
+ * to javascript. It was translated when their main branch was at their
+ * [acbe764fb7f12c7a8b0696156283fbcb5073fcd7 commit](https://github.com/alphagov/notifications-utils/tree/acbe764fb7f12c7a8b0696156283fbcb5073fcd7).
+ * The comments below that refer to files and line numbers are relative
+ * to that commit.
+ *
+ * The only differences between this file and theirs are the translation
+ * from python to javascript and some tidying to match our linter rules.
+ */
+
+
+/**
+ * Formats the postcode for regex checking by removing whitespace, obscure whitespace and
+ * converting it to uppercase.
+ *
+ * Originally found in `notifications_utils/postal_address.py#L150-L151`, at
+ * the above repo.
+ *
+ * @param {string} postcode The postcode entered by the candidate
+ * @returns {string} normalisePostcode The postcode after being reformated with whitespace removed
```

Comment:
The correct format for `@returns` lines is 
```js
/**
* ...
* @returns {string} The postcode after being reformated with whitespace removed
*/
```

You don't include the variable name as it's not relevant to the calling code.

Reply:
Resolved in 362cf94ef25e4e3d2533f52fdb8f33ebde524002.